docs/proposals/20191030-machine-health-checking.md

```diff
@@ -158,20 +164,25 @@ type target struct {
 
 - Calculate the number of unhealthy targets.
 - Compare current number against `maxUnhealthy` threshold and temporary short circuits remediation if the threshold is met.
-- Trigger remediation for unhealthy targets i.e request machines for deletion.
+- Annotates unhealthy target machines as described above.
 
 Out of band:
-- The owning controller e.g machineSet controller reconciles to meet number of replicas and start the process to bring up a new machine/node.
+- The owning controller observes the annotation and is responsible to remediate the machine.
 - The machine controller drains the unhealthy node.
 - The machine controller provider deletes the instance.
-- The machine controller deletes the machine.
+- The owning controller MUST delete the machine.
```

Comment:
I realise this maintains the original ordering, but if this list is ordered, the owning controller deleting the machine triggers the drain and instance delete so shouldn't it come first?

Might also be worth adding something along the lines of `The owning controller performs any pre-deletion tasks required` to this list?